app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Done.kt

```diff
@@ -0,0 +1,70 @@
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import cm.aptoide.pt.aptoide_ui.theme.green
+
+@Preview
+@Composable
+fun TestDoneIcon() {
+  Image(
+    imageVector = getDoneIcon(),
+    contentDescription = null,
+    modifier = Modifier.size(64.dp)
+  )
+}
+
+fun getDoneIcon(): ImageVector =
+  ImageVector.Builder(
+    name = "DoneIcon",
+    defaultWidth = 64.dp,
+    defaultHeight = 64.dp,
+    viewportWidth = 64f,
+    viewportHeight = 64f,
+  ).apply {
+    path(fill = SolidColor(Color(0xFF25B47E))) {
```

Comment:
Let's not hardcode here but provide it through arguments.
Better set one of the new colours in the place it is used.

Reply:
success views have a different ticket. I just added them for the project to compile